pipeline-runner/data-raw/sysdata.R

```diff
@@ -0,0 +1,9 @@
+# constrants used in GEM2S
+gem2s <- list(
+    max.edrops.fdr = 0.001,
+    max.empty.counts = 100,
+    max.empty.drops = 50
+)
```

Comment:
consider adding the cellranger_fnames from file `gem2s-1-download_cellranger_files`, line 17 because these are also configuration, as well as this `/input` dir name used in that file and elsewhere